arch/x86/kernel/traps.c

```diff
@@ -1443,5 +1444,8 @@ void __init trap_init(void)
 	if (!cpu_feature_enabled(X86_FEATURE_FRED))
 		idt_setup_traps();
 
+	/* If SKINIT was done on the BSP, this is the spot to enable GIF */
```

Comment:
According to `Software Requirements for Secure MP initialization` in 24593 AMD APM Vol.2, SKINIT can be invoked on BSP only, the way this comment is written may suggest otherwise. Maybe something like:

```suggestion
	/* On the BSP, this is the spot to enable GIF if SKINIT was done  */
```

Reply:
https://github.com/TrenchBoot/linux/compare/bfc6b41c8d4bdebc6eab7a762f03f0010ab06f0e..1f29b6db37352ae156ab2e1a2cd62e7e050659cf#diff-51ce909c2f65ed9cc668bc36cc3c18528541d8a10e84287874cd37a5918abae5